dynamic/main.go

```diff
@@ -94,6 +95,34 @@ func initialSetup() (info.Provider, pfbridge.ProviderMetadata, func() error) {
 				info.SchemaPostProcessor(&packageSchema.PackageSpec)
 			}
 
+			if fullDocs {
```

Comment:
Why are we associating "generate a full schema" with "generate a docs page"? Can we allow them to be set independently?
```suggestion
			if indexDocsOutDir != "" {
```

Reply:
This is meant as an internal feature to generate docs for the registry, where my expectation is we'd always want to do it together.
I almost added it as a default part of the `fullDocs` flag originally, but wanted to be able to specify an outDir (and not write to a physical file if outDir is unspecified).